### PR TITLE
add hint for JACK_DEFAULT_SERVER

### PR DESCRIPTION
**Pull request by djfun**
See discussion in #1397

See jack2 source code for comparison:
https://github.com/jackaudio/jack2/blob/a9c362060ba4182e3ea61a8d142d45da9ef63762/common/varargs.h#L44

### CODE REVIEW

File: linux/sound.cpp [hunk 1]
```diff
@@ -31,6 +31,13 @@ void CSound::OpenJack ( const bool  bNoAutoJackConnect,
                         const char* jackClientName )
 {
     jack_status_t JackStatus;
+    const char *serverName;
+
+    if ((serverName = getenv("JACK_DEFAULT_SERVER")) == NULL) {
```
Comment (hoffie) [inline linux/sound.cpp:36]: With the rather spacing-centric coding style of Jamulus, it should probably look like this :)

```suggestion
    if ( ( serverName = getenv( "JACK_DEFAULT_SERVER" ) ) == NULL ) {
```

**Review by hoffie — COMMENTED** [review]
Seems good, logic-wise. Works properly.

It currently looks like this:
```
- connect on startup to address: localhost
- initialization file name: /tmp/x
Connecting to jack "default" instance (use the JACK_DEFAULT_SERVER environment variable to change this).
```

Should it also have a leading "- " similar to the other, status messages?

**Comment by pljones** [comment]
> Should it also have a leading "- " similar to the other, status messages?

The "-" is leading on command line option outputs.  This is emitted from the linux sound code on start up.  So, not necessarily.  (I base this on how the jam recorder works.)

File: linux/sound.cpp (hunk 1 above)
Comment (djfun) [reply linux/sound.cpp]: I added the changes and force-pushed a new version.

**Review by hoffie — APPROVED** [review]
Thanks!

**Review by pljones — APPROVED** [review]

**Pull request merged**